nifi-nar-bundles/nifi-jslt-bundle/nifi-jslt-processors/src/main/java/org/apache/nifi/processors/jslt/JSLTTransformJSON.java

```diff
@@ -277,4 +329,36 @@ private String readTransform(final PropertyValue propertyValue) {
             throw new UncheckedIOException("Read JSLT Transform failed", e);
         }
     }
-}
+
+
+    protected JsonNode getNextJsonNode(final String transformStrategy) throws IOException {
```

Comment:
This is where it would be helpful to use the `enum` value for the selected strategy.